### ISSUE TITLE
Disable v-wave on "disabled" state for buttons

### ISSUE THREAD

**Issue by tassin-gauthier** [justintaddei/v-wave#543]
Hello, I think adding the possibility to disable v-wave on "disabled" state when the v-wave directive is applied to the buttons could be great. I don't know if it should be a default behavior because it could introduce a breaking change, but maybe add a boolean to the global config ?

I know it's currently possible to do it by copy / paste the same code used into "disabled" button attribute, but it's a lot of duplication, especially when you get many buttons

**Comment by justintaddei** [comment]
Love the idea! I'll look into it!

**Comment by justintaddei** [comment]
Purposed option:


| name                         | default | type        |
|------------------------|---------|-----------|
| respectDisabledAttr  | false     | boolean  |

Wave will be disabled if the disabled attribute is present on the button (can be set globally):
```vue
<button disabled v-wave="{
  respectDisabledAttr: true
}">Click me!</button>
```

> Ideally this would've been included with the 2.0.0 release and defaulted to `true` but I didn't want to delay Nuxt 3 support any longer than it already had been.

**Comment by JaxThePrime** [comment]
@justintaddei Im using the directive like this and doesnt work for me, there is no description about this attr and usage in the documentation

import VWave from "v-wave";

const { vWave } = VWave.createLocalWaveDirective({
  respectDisabledAttr: true
});
